apps/api-server/src/routes/api/project.js

```diff
@@ -75,16 +75,39 @@ router.route('/')
 	.get(auth.can('Project', 'list'))
 	.get(pagination.init)
 	.get(function(req, res, next) {
+    if (req.query.includeAuthConfig) return next('includeAuthConfig is not implemented for projects list')
+    return next();
+	})
+	.get(async function(req, res, next) {
+
+    try {
+      let where = {};
+      if (!hasRole( req.user, 'superuser' )) {
+        // first find all corresponding users for the current user, only where she is admin
+        let users = await db.User.findAll({
+          where: {
+            idpUser: {
+              identifier: req.user?.idpUser?.identifier || 'no identifier found',
+              provider: req.user?.idpUser?.provider || 'no provider found',
+            },
+            role: 'admin',
+          }
+        })
+        let projectIds = users.map(u => u.projectId);
+        where = { id : projectIds };
+      }
+
+      // now find the corresponding projects
+		  let result = await db.Project.scope(req.scope).findAndCountAll({ offset: req.dbQuery.offset, limit: req.dbQuery.limit, where })
+      req.results = result.rows;
```

Comment:
```suggestion
      // now find the corresponding projects
      let result = await db.Project.scope(req.scope).findAndCountAll({ offset: req.dbQuery.offset, limit: req.dbQuery.limit, where })
      req.results = result.rows;
```

Deze sprong er toevallig uit